PocketKit/Sources/PocketKit/Home/HomeViewController.swift

```diff
@@ -63,14 +69,14 @@ class HomeViewController: UIViewController {
         super.viewDidLoad()
 
         Task {
-            let slates = try? await source.fetchSlates()
-            setSlates(slates: slates)
+            let lineup = try? await source.fetchSlateLineup(Self.lineupID)
+            self.slateLineup = lineup
```

Comment:
this should either call `setSlateLineup` or we should get rid of that function. I'm really not sure if the setSlateLineup function is necessary. I had only put it so that we could execute that code on the main thread. However, I'm pretty sure that `Task {}` will resume on the thread it was originally started on so it may not be necessary.